### PR TITLE
[MAINTENANCE] AlertControl tests

### PR DESCRIPTION
**Pull request by NathanFarmer**
- Test that alerts render

### CODE REVIEW

File: src/stories/controls/AlertControl.test.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          {
+            type: "Label",
+            text: "An African swallow, maybe -- but not a European swallow, that's my point.",
+            options: {}
+          }
+        ],
+      },
+  });
+  const input = await screen.findByRole("alert");
+  expect(input).toHaveClass("ant-alert-info");
```
Comment (DrewHoo) [inline src/stories/controls/AlertControl.test.tsx:37]: you can omit this expect.
- we don't control antd classnames and they don't show up in our code, so i don't like writing tests based on those classnames
- `findBy*` will reject (and the test will fail) if it doesn't find the alert by its aria role, so it is sufficient by itself

Comment (NathanFarmer) [reply src/stories/controls/AlertControl.test.tsx:37]: The intent of this test was to assert that the default mode whe not passed `options` is "info". What you propose it more of a "does it render" test then right?

File: src/stories/controls/AlertControl.test.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+});
+
+
+test.each([
+  ["info", "ant-alert-info"],
+  ["warning", "ant-alert-warning"],
+  ["success", "ant-alert-success"]
+])("label option type sets antd class", async (optionType: string, className: string) => {
```
Comment (DrewHoo) [inline src/stories/controls/AlertControl.test.tsx:45]: I think you can type optionType as `AlertLabelOptions` from ui-schema.ts

Comment (DrewHoo) [reply src/stories/controls/AlertControl.test.tsx:45]: I also wonder if we could adjust that type to reflect the antd Alert's `type` property type, and omit this test bc static typechecking would then always fail before this test could fail

File: src/ui-schema.ts [hunk 3]
```diff
@@ -116,18 +117,16 @@ interface LabelDescription {
...
  * A label element.
  */
-interface LabelElement extends UISchemaElement, Internationalizable {
+export interface LabelElement extends UISchemaElement, Internationalizable {
   type: "Label"
   /**
    * The text of label.
    */
   text: string
-  options: LabelOptions
```
Comment (NathanFarmer) [inline src/ui-schema.ts:125]: Keeping `options` as optional is consistent with `UISchemaElement`.

**Review by DrewHoo — APPROVED** [review]
Nice work!

**Pull request merged**